1-mandatory/1-writers.js

```diff
@@ -72,7 +79,11 @@ Exercise 3:
 */
 
```

Comment:
Similar comment to above.
The way you have done it is fine - but sometimes a few small and simple array methods can make your easier to read.